src/lib/detection_algorithms/yolo/main.js

```diff
@@ -0,0 +1,2 @@
+module.exports.Exporter = require('./exporter.js').Exporter
```

Comment:
So for the change which removes the config.json you'll have here 
module.exports.displayName = 'YOLO';